scripts/tests.sh

```diff
@@ -52,17 +52,32 @@ cleanup() {
 }
 trap cleanup INT TERM
 
-if [ "$1" = --junit_report ]
-then
-    if [ -z "$2" ]
-    then
-        echo "Usage: $0 [--junit_report <report_directory>]"
-        exit 1
-    fi
-    log_directory="$2"
-else
-    log_directory=""
-fi
+log_directory=""
+no_smt=""
+while [[ $# -gt 0 ]]
+do
+    case "$1" in
+        --junit_report)
+            if [ -z "$2" ]
+            then
+                echo "Usage: $0 [--junit_report <report_directory>] [--no-smt]"
```

Comment:
Just noticed this. Does `$0` still hold the program name after you use `shift`?

Reply:
yeah, only arguments are `shift `ed in the loop.